site2/docs/admin-api-brokers.md

```diff
@@ -128,7 +128,7 @@ BrokerInfo(serviceUrl=broker1.use.org.com:8080)
 </TabItem>
 <TabItem value="REST API">
 
-{@inject: endpoint|GET|/admin/v2/brokers/leaderBroker?version=@pulsar:version_number@}
+{@inject: endpoint|GET|/admin/v2/brokers/leaderBroker|operation/getLeaderBroker?version=@pulsar:version_number@}
```

Comment:
1. Do these links go to the specific REST API doc rather than the home page?
2. Will you fix all the occurrences in `master`?

Reply:
1. Yes. The logic has been verified. Theoretically, they can go to the correct place but this cannot be directly verified in the local build because what we use for version numbers is a variable. 
2. I've corrected all the occurrences that I've searched out in the master branch.

Reply:
for issue 2, I mean we have many REST API links in the whole Pulsar doc, will you fix them all (on `master`)?

Example:
<img width="1377" alt="image" src="https://user-images.githubusercontent.com/50226895/176585165-600f2435-d473-4eb8-9c0d-1e7f71bdef34.png">

Reply:
Oh, I see your point. I didn't expect we have such cases. The search term I use is "@pulsar:version_number@", to make sure it is injected in the correct place. So these cases are not included. I can resolve them in another PR after verifying the results of this PR on the Pulsar website.

Reply:
After looking to the md file, I found the link itself is correct, which contains "@Pulsar:version_number@" and the variable is replaced with the version number of docs as we can see in the website. The only issue is: it is written in a command style, rather than a link.

Reply:
OK, thanks!

Reply:
@Anonymitaet the updates for the cmd style have been committed.